### DISCUSSION TITLE
Add number of contacts to "Contacts" button

### DISCUSSION THREAD

**Discussion by manelrodero** [simple-login/app#1914] [Feature Requests]
Idea to quickly see the contacts that have had an alias.

![image](https://github.com/simple-login/app/assets/26098260/f0d63ef6-a6f2-4208-95ab-3806dc09cbb2)

If it has zero, it is likely that you have not registered it anywhere and its deletion could be considered.

If it has 1 or more, these give you a clue as to where you registered it.

**Comment by gfleetwood** [comment]
Piggy backing on this request can we get a way to view the unique list of contacts for an alias? The contacts as it is now shows all mail from an address, but I'd like a way to see that this one address is a contact. That would also be great for this feature request because you could then have a unique count of contacts -  and the relevant view - vs a count of all the mail sent to the alias.
